components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -492,30 +493,21 @@ private void deleteDELETEOperationsTask(Timestamp timestamp) {
             log.error(e.getMessage(), e);
             return;
         }
+        long cleanupLimit = TimeUnit.MINUTES.toNanos(FrameworkUtils.getCurrentStandardNano() - IdentityUtil
```

Comment:
Do the same fix as above,

long cleanupLimitNano = FrameworkUtils.getCurrentStandardNano() - TimeUnit.MINUTES.toNanos(IdentityUtil.getCleanUpTimeout()));